pallas-network/src/multiplexer.rs

```diff
@@ -63,6 +67,7 @@ pub struct Segment {
 #[cfg(target_os = "windows")]
 pub enum Bearer {
     Tcp(TcpStream),
+    Socket(NamedPipeClient)
```

Comment:
could we use `NamedPipe` for the enum member instead of `Socket`?
because I'm afraid that `Socket` can be misinterpreted